app/controllers/articles_controller.rb

```diff
@@ -2,6 +2,7 @@ class ArticlesController < ApplicationController
 
   http_basic_authenticate_with name: "dhh", password: "secret",
                                except: [:index, :show]
+
   def index
     @articles = Article.all
```

Comment:
You don't need to fix it now, but for future understanding: it's dangerous case when you request all records from a table at once. It could be millions of records in one table and if you'll try to query them all at once, 99% probability your app will die due to lack of memory. In 99.99% cases you'll need to either filter records, or implement pagination (gems `kaminari`, `will_paginate` etc.)